tests/overdraft/request_1_2821_create_account_1.json

```diff
@@ -3,12 +3,12 @@
   "H": {
     "version": "",
     "created": "2018-09-27T16:26:22.830373928+09:00",
-    "signature": "5TVx3JQHgZhHGcNLAyLfJQfssu4Hh7QRWBfjymEd7i1yC2s3yjSjKs92nwdkupKjFsQdbUNZHdkSQhicrT8ZSVWz"
+    "signature": "44unZEoQy41vYQr8LCFwJfXv8FpM6waxddwDd4sAs7Koo1XiJy18Lvp2FH6JG18DbvMwJTs5VQnoVoNEyJdZqicM"
   },
   "B": {
     "source": "GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ",
     "fee": "10000",
-    "sequenceID": 0,
+    "sequenceid": 0,
```

Comment:
Why is this changed ?

Reply:
I think it caused when I print the body and signature with [sebak-tx-sign](https://github.com/spikeekips/sebak-tools/tree/master/sebak-tx-sign). Indeed, there was a reason.
BTW, it does not affect the signature.

Reply:
And sebak deserialize it okay, I suppose ? Anyway, since it has nothing to do with changing that algorithm, it should not have been changed here.

Reply:
> And sebak deserialize it okay, I suppose ?

I think so. Yes, it was my mistake. I was too happier when integration tests are passed. If it causes any problems, it can be reverted.